### PR TITLE
chore: fix function names in comment

### PR DESCRIPTION
**Pull request by racequite**
**- What I did**

fix function names in comment

**- How I did it**

**- How to verify it**

**- Description for the changelog**

```markdown changelog

```

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by vvoland** [comment]
Thanks!

We require all commit messages to have a `Signed-off-by` line with your actual name (not a pseudonym)
and e-mail (see ["Sign your work"](https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work)
in the CONTRIBUTING.md in this repository), which looks like:

```
Signed-off-by: YourFirsName YourLastName <yourname@example.org>
```!

File: client/client_test.go [hunk 1]
```diff
@@ -340,7 +340,7 @@ func TestNegotiateAPIVersion(t *testing.T) {
 	}
 }
 
-// TestNegotiateAPIVersionOverride asserts that we honor the DOCKER_API_VERSION
+// TestNegotiateAPVersionOverride asserts that we honor the DOCKER_API_VERSION
 // environment variable when negotiating versions.
 func TestNegotiateAPVersionOverride(t *testing.T) {
```
Comment (coolljt0725) [inline client/client_test.go:345]: I think we should change `TestNegotiateAPVersionOverride` to `TestNegotiateAPIVersionOverride `  to make the function name more readable here.  Same on line https://github.com/moby/moby/pull/47731/files#diff-57cdc3e29d0ef725a50897a4fbd1c4f76353429fa66d3a785dcbc17dcd7b6f31R359

**Comment by coolljt0725** [comment]
@racequite  sorry for the delay. Looks we also have similar issues in https://github.com/moby/moby/blob/master/client/client_test.go  like:
https://github.com/moby/moby/blob/ae976b998be20312526fb0c1bfc4ef4c9d6a19d4/client/client_test.go#L345
https://github.com/moby/moby/blob/ae976b998be20312526fb0c1bfc4ef4c9d6a19d4/client/client_test.go#L357 
https://github.com/moby/moby/blob/ae976b998be20312526fb0c1bfc4ef4c9d6a19d4/client/client_test.go#L359,
In these places, the `AP` should be `API`
could you please have these fixed in this PR as well ?

**Comment by thaJeztah** [comment]
Let me bring this one in; I'll open a quick follow-up for those other ones 👍

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
- opened https://github.com/moby/moby/pull/47937

**Pull request merged**